+incf/+livescripts/setup.m

```diff
@@ -39,8 +39,8 @@
 
     switch from_location
         case "online"
-            % TODO
-
+            msgbox("Unfortunately, MATLAB Online is not yet automated." + newline + ...
+                   "Please use the 'local' location and manually create/select a folder in your MATLAB Drive.");
```

Comment:
Suggest to say "option" instead of "location" to make it clear you're referring to the dropdown options

Reply:
@vijayiyer05 Thanks a lot for the review! I updated the PR accordingly